runners/apex/pom.xml

```diff
@@ -215,6 +215,7 @@
               <ignoredUsedUndeclaredDependencies>
                 <ignoredUsedUndeclaredDependency>org.apache.apex:apex-api:jar:3.5.0-SNAPSHOT</ignoredUsedUndeclaredDependency>
                 <ignoredUsedUndeclaredDependency>org.apache.commons:commons-lang3::3.1</ignoredUsedUndeclaredDependency>
+                <ignoredUsedUndeclaredDependency>commons-io:commons-io:jar:2.1</ignoredUsedUndeclaredDependency>
```

Comment:
These are transitive Apex dependencies that are used by the runner implementation. Entries here are used instead of repeating the dependency, as the intention is to use the transitive dependency at the exact version.